### ISSUE TITLE
MO 1.3.11 nxmhandler.exe spam

### ISSUE THREAD

**Issue by Astor1977** [TanninOne/modorganizer#925]
I clicked the little blue planet icon, which took me to Nexus where i found a mod i wanted to download.  Only I don't get the mod anymore, now my Task Manager is filled with nxmhandler.exe opening over and over until my PC completely locks up.
 
I reinstalled MO and that helped for a day, now it's doing it again.   I can still manually download mods and install them with MO, so i'm not completely buggered; just confused and slightly annoyed.
 
Help?

**Comment by Pathal** [comment]
I'm getting the same issue in 2.0.7, using Chrome to browse Nexus.

`Taskkill /IM nxmhandler.exe /F`

Anyone else with this issue can use that to prune the processes if you run into the problem in the meantime, but once they start overflowing into the page file getting into cmd is not really an option and you'll have to reboot.

**Comment by Gernash** [comment]
Is Mod organizer  and your in base bod directory in your defender exclusions

@Pathal 
http://forum.step-project.com/topic/11094-wip-fallout-4-using-modorganizer-2/

**Comment by Pathal** [comment]
Yeah they were. And I run multiple versions of Mod Organizer, one for each game. I had the problem with 1.3.11 with Skyrim, MO2 b2 with FO4, and I tried updated to a single copy of 2.0.7 and removing the other clients to try and figure out a solution. It's still happening. Now I'm wondering if it had to do with that win10 anniversary update.

**Comment by Gernash** [comment]
I'm running MO2 for FO4 via wiki it's stable and solid
I'm running MO 1.3.11 for Skyrim it's also solid

the only thing is you have to have one ot the other open before you click the managed download or it crashes the mxhandler

**Comment by Pathal** [comment]
> the only thing is you have to do is have one or the other open before you click the managed download or it crashes the mxhandler

Lets tone it back a little. I'm not that tech illiterate. My clients were working fine until recently, hence why I'm thinking the anniversary update messed something up.

What operating system are you on?

**Comment by Gernash** [comment]
Windows 10 Pro
"one or the other open"
skyrim open mo 1.3.11 then use the Managed downlod.
FO4 open mo 2 then use the Managed downlod.
if it still hangs then it's a MX nxmhandler.exe issue

From the Guide I linked 
Notice: To resolve download manager issue from Nexusmod site I recommend deleting the nxmhandler.ini from C:\ModOrganizer2 and C:\ModOrganizer2Data then run nxmhandler.exe in C:\ModOrganizer2 folder to generate the nxmhandlers.ini files

**Comment by Gernash** [comment]
Or wherever you have installed them

**Comment by Pathal** [comment]
ini file isn't there. It's %AppData%\Local\ModOrganizer. It didn't fix the issue.

However wiping the rest of the settings of MO 2.0.7 does fix it. Unfortunately that means I'm out of my download lists but there's probably some what I can do that myself. Also have no idea if this problem will return like it did for Astor.

**Comment by Gernash** [comment]
If you are using it in PORTABLE MODE then it's in your root MO2 folder

**Comment by Gernash** [comment]
Follow the guide if your reinstalling your life will be better and Astor's issue is similar needs to go in MO1.3.11 folder and run the mxhandler to recreate the link or go into setting and recreate the association (this button works with mo1.3.11)
